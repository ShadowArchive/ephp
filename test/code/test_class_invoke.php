<?php

class A {
    public function __invoke() {
        print "Hello world!\n";
    }
}

class B extends A {}
class C {}

$a = new A;
$a();

$b = new B;
$b();

$c = new C;
$c();

