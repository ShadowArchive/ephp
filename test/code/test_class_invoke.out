Hello world!
Hello world!

Fatal error: Function name must be a string in {{CWD}}/test/code/test_class_invoke.php on line 19
